### PR TITLE
Add minimal vscode settings for md files

### PR DESCRIPTION
**Pull request by bogdandrutu**
Similar with https://github.com/open-telemetry/opentelemetry-specification/blob/master/.vscode/settings.json

### CODE REVIEW

**Comment by bogdandrutu** [comment]
@open-telemetry/specs-approvers please review :)

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "rewrap.wrappingColumn": 80,
+    "editor.rulers": [80],
+    "markdownlint.config": {
+        "MD004": false,
```
Comment (dyladan) [inline .vscode/settings.json]: Will this list of exceptions need to be maintained indefinitely?

Comment (bogdandrutu) [reply .vscode/settings.json]: We can update them when they are not required.

Comment (bogdandrutu) [reply .vscode/settings.json]: I copied the file from the specs

**Comment by carlosalberto** [comment]
@bogdandrutu I suggest we merge this one - after all, this is not an actual OTEP, but changes to this repo itself, and doesn't need that many approvals.

**Comment by bogdandrutu** [comment]
@carlosalberto ok

Using the admin power and merging this with only 3 approvers

**Pull request merged**